### PR TITLE
Fix jenkins URLs for new buildvm

### PR DESCRIPTION
**Pull request by vfreex**

### CODE REVIEW

**Review by joepvd — APPROVED** [review]
/lgtm

**Review by locriandev — DISMISSED** [review]
All other functions got added a new param `jenkins_url`, while this one is using an env var? Is there a reason for this? Maybe we could have a unified approach across the whole module

**Review by joepvd — APPROVED** [review]
/lgtm

File: pyartcd/pyartcd/jenkins.py [hunk 1]
```diff
@@ -361,8 +366,9 @@ def update_title(title: str, append: bool = True):
     """
 
     job = jenkins_client.get_job(current_job_name)
+    jenkins_url = os.environ["JENKINS_URL"] or constants.JENKINS_SERVER_URL
     build = Build(
-        url=current_build_url.replace(constants.JENKINS_UI_URL, constants.JENKINS_SERVER_URL),
+        url=current_build_url.replace(constants.JENKINS_UI_URL, jenkins_url),
```
Comment (joepvd) [inline pyartcd/pyartcd/jenkins.py:371]:
```suggestion
        url = current_build_url.replace(constants.JENKINS_UI_URL, jenkins_url),
```

File: pyartcd/pyartcd/jenkins.py [hunk 2]
```diff
@@ -387,8 +393,9 @@ def update_description(description: str, append: bool = True):
...
```
Comment (joepvd) [inline pyartcd/pyartcd/jenkins.py:398]:
```suggestion
        url = current_build_url.replace(constants.JENKINS_UI_URL, jenkins_url),
```

File: pyartcd/pyartcd/jenkins.py [hunk 3]
```diff
@@ -32,19 +32,23 @@ class Jobs(Enum):
     CINCINNATI_PRS = 'aos-cd-builds/build%2Fcincinnati-prs'
 
 
-def init_jenkins():
+def init_jenkins(jenkins_url: Optional[str]=None):
```
Comment (thegreyd) [inline pyartcd/pyartcd/jenkins.py:35]:
```suggestion
def init_jenkins(jenkins_url: Optional[str] = None):
```

File: pyartcd/pyartcd/jenkins.py [hunk 4]
```diff
@@ -361,8 +366,9 @@ def update_title(title: str, append: bool = True):
     """
 
     job = jenkins_client.get_job(current_job_name)
+    jenkins_url = os.environ["JENKINS_URL"] or constants.JENKINS_SERVER_URL
     build = Build(
-        url=current_build_url.replace(constants.JENKINS_UI_URL, constants.JENKINS_SERVER_URL),
+        url = current_build_url.replace(constants.JENKINS_UI_URL, jenkins_url),
```
Comment (thegreyd) [inline pyartcd/pyartcd/jenkins.py:371]:
```suggestion
        url=current_build_url.replace(constants.JENKINS_UI_URL, jenkins_url),
```

File: pyartcd/pyartcd/jenkins.py [hunk 5]
```diff
@@ -387,8 +393,9 @@ def update_description(description: str, append: bool = True):
...
```
Comment (thegreyd) [inline pyartcd/pyartcd/jenkins.py:398]:
```suggestion
        url=current_build_url.replace(constants.JENKINS_UI_URL, jenkins_url),
```

**Pull request merged**